.github/workflows/ci-client-cyclic-deps-check.yml

```
@@ -55,51 +55,70 @@ jobs:
         run: |
           yarn install --immutable
 
-      - name: Count circular dependencies on release branch
-        id: count-cyclic-deps-in-release
+      - name: Count circular dependencies on base branch
+        id: count-cyclic-deps-in-base
         if: steps.changed-files.outputs.any_changed == 'true'
         run: |
-          dpdm "./src/**/*.{js,jsx,ts,tsx}" --circular --warning=false --tree=false > release_circular_deps.txt
-          # awk 'NF' release_circular_deps.txt: Filter out empty lines from the file
+          npx dpdm "./src/**/*.{js,jsx,ts,tsx}" --circular --warning=false --tree=false \                     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)